msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -29,27 +32,25 @@ You can't test this feature during development. App customization isn't supporte
 
 Provide guidelines for customers (specifically Teams admins) who want to customize your app. For more information, see [customize apps in Teams](/MicrosoftTeams/customize-apps).
 
-## Hide Teams app until admin approves
+## Hide Teams app from users
```

Comment:
Incorrect title. It may give an impression that there are such a thing as hidden apps. Users may incorrectly think that this app is hidden from Store view but can be accessed via, say a secret link. (Think something on the lines of unlisted videos on YouTube or file sharing only with people who have a link.)